mlos_bench/mlos_bench/environments/local/local_env.py

```diff
@@ -240,6 +240,7 @@ def _datetime_parser(datetime_col: pandas.Series) -> pandas.Series:
         ValueError
             On parse errors.
         """
+        # FIXME: We should probably not be assuming results arrived in UTC timezone.
         new_datetime_col = pandas.to_datetime(datetime_col, utc=True)
```

Comment:
This is the main problem with the remaining two sets of test cases that are failing.

Essentially, this conversion happens incorrectly, so we need to handle it a little differently.

Here's an example:
```sh
TZ="America/Chicago" pytest -n0 mlos_bench/mlos_bench/tests/environments/local/local_env_telemetry_test.py
```

Reply:
Will take a look at this more tomorrow.

Reply:
NOTE: that form of testing doesn't currently happen in the pipeline - will have to fix that later too.

Reply:
The root cause is also fixed now.